tests/e2e/default-mobile/reports/search.wdio-spec.js

```diff
@@ -35,6 +36,8 @@ describe('Search Reports', () => {
     await utils.saveDocs([ ...places.values(), districtHospitalPatient, healthCenterPatient ]);
     reportDocs = await utils.saveDocs(reports);
     await loginPage.cookieLogin();
+    // Delete training doc to avoid interference with search results
+    await utils.deleteDoc(constants.DEFAULT_USER_ADMIN_TRAINING_DOC._id);
```

Comment:
Can this deletion be done in the `login` helper?
```
 await loginPage.cookieLogin({ deleteTraining: true });
```
This way we don't need tests to load constants and do this explicitly.